libcaja-private/caja-file-operations.c

```diff
@@ -4237,6 +4237,10 @@ copy_move_file (CopyMoveJob *copy_job,
 		goto out;
 	}
 
+	/* Timer was already running from g_timer_new()...
+	 * Restart now that we are actually moving or copying
+	 */
+	g_timer_start( job->time );
```

Comment:
Fix Code Formatting:
```patch
diff --git a/libcaja-private/caja-file-operations.c b/libcaja-private/caja-file-operations.c
index 8079cd9..8462d22 100644
--- a/libcaja-private/caja-file-operations.c
+++ b/libcaja-private/caja-file-operations.c
@@ -4240,7 +4240,7 @@ copy_move_file (CopyMoveJob *copy_job,
        /* Timer was already running from g_timer_new()...
         * Restart now that we are actually moving or copying
         */
-       g_timer_start( job->time );
+       g_timer_start (job->time);
 
  retry:
        error = NULL;
```

Reply:
Whoops! Not sure where my head was, usually I'm better at picking up near-by style :-)

Fixed in 4ec34f7